packages/metadata-tools/src/config-metadata.ts

```diff
@@ -0,0 +1,286 @@
+import type { OmniPointHardhat, OmniEdgeHardhat } from '@layerzerolabs/devtools-evm-hardhat'
+import type { OAppEnforcedOption, OAppEdgeConfig } from '@layerzerolabs/ua-devtools'
+
+const METADATA_URL = process.env.LZ_METADATA_URL || 'https://metadata.layerzero-api.com/v1/metadata'
+
+interface IMetadata {
+    [key: string]: {
+        created: string
+        updated: string
+        tableName: string
+        environment: string
+        blockExplorers?: { url: string }[]
+        deployments?: {
```

Comment:
Just mentioning that we may want to split this into a union of EndpointV1 and EndpointV2 facets.  For now, I am fine with keeping this typing;  we should ask the offchain team if they plan to publicize types for public apis.

Reply:
Asked. Publishing public types package for metadata in something they plan. We can come back to this typing issue after the package is released. Our efforts should probably focus on expediting that process instead of maintaining our own types so for now I'll mark this conversation as resolved.